README.md

```diff
@@ -57,10 +57,9 @@ sudo apt update && sudo apt install -y dnsutils
 Create a directory to store SimpleLogin data:
 
 ```bash
-mkdir sl
-mkdir sl/pgp # to store PGP key
-mkdir sl/db # to store database
-mkdir sl/upload # to store quarantine emails
+mkdir -p /sl/pgp # to store PGP key
```

Comment:
I'm not sure using `/sl`, `/dkim.key`, etc as paths will suit most users as they might want to run SimpleLogin with the least privileges. I'd recommend to not change the paths for these files as we might need to change the commands for upgrading SimpleLogin as well (in the https://github.com/simple-login/app/blob/e5a7aeb3fb0db89707212c75315dd7c8401f386b/docs/upgrade.md#L1-L0)

Reply:
I originally had it as your version but SimpleLogin database failed after installing on a Linux OS where root is the default user. The login page loads but when you enter credentials it says an error occured. I then tried installing to `/opt/sl` as a regular user but your Docker images weren't compatible with that path.

Reply:
Sure, I'll restore those path edits but I'm curious if your install using those instructions survives a server reboot?

Reply:
Yes they work after a server reboot. I don't see why they wouldn't work afer a reboot though.